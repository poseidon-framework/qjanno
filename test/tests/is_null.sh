qjanno "SELECT * FROM data/csv/basic.csv WHERE bar IS NULL"
