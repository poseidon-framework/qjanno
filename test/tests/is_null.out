c0  c2
